core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -800,7 +802,7 @@ private Name() {} // prevent instantiation
    * @return whether the input is a valid property name
    */
   public static boolean isValid(String keyStr) {
-    return KEYS_MAP.containsKey(keyStr);
+    return KEYS_MAP.containsKey(keyStr) || keyStr.startsWith("fs");
```

Comment:
Just to double check with you, how many new properties we are introducing here? I only see `STOCATOR_SCHEME_LIST`. If you have more that you want to support for this integration, you should add them to `PropertyKey`